src/js/components/DataFilter/__tests__/DataFilter-test.tsx

```diff
@@ -201,6 +203,54 @@ describe('DataFilter', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('range prop step', async () => {
+    const { asFragment } = render(
+      <Grommet>
+        <Data data={data}>
+          <DataFilters>
+            <DataFilter
+              property="total"
+              range={{ step: 15, min: 0, max: 250 }}
+            />
+          </DataFilters>
+        </Data>
+      </Grommet>,
+    );
+
+    const lowerBound = screen.getByRole('slider', { name: 'Lower Bounds' });
+    fireEvent.mouseDown(lowerBound);
+    fireEvent.mouseMove(lowerBound, { clientX: 31, clientY: 20 });
+    fireEvent.mouseUp(lowerBound);
+    expect(lowerBound.getAttribute('aria-valuenow')).toEqual('45');
+    expect(asFragment()).toMatchSnapshot();
```

Comment:
Do we need a snapshot test here or can we just rely on the check above `expect(lowerBound.getAttribute('aria-valuenow')).toEqual('45');`?

Reply:
The snapshot is just capturing styling info, while the previous expect is more so testing fuctionailty.

Given there are other snapshots capturing range, should I remove this one to keep things more lean?

Reply:
Yeah let's remove the snapshot. This article has some good points about snapshot testing: https://kentcdodds.com/blog/effective-snapshot-testing. My main takeaway was that snapshot testing can be very useful, but I think in Grommet we have a tendency to overuse it a bit when the test could be written using other assertions.

Reply:
Totally agree, I'll remove.